Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -1826,8 +1826,10 @@ static IntRect computeMarqueeSelectionRect(const WebCore::IntPoint& point1, cons
     notifySelectionChanged();
 }
 
-String UnifiedPDFPlugin::getSelectionString() const
+String UnifiedPDFPlugin::selectionString() const
 {
+    if (!m_currentSelection)
```

Comment:
Do we really need this? My understanding is that ObjC doesn't need null checks.

Reply:
That's correct. I originally wrote this method without the null check, but after a conversation with some of the other folks working on the PDF plugin, I decided that it might be better to do an explicit check anyway for readability's sake. It gets rid of the need to reason about whether or not we're accessing a C++ struct or messaging an ObjC item.